packages/opencv-python/test_opencv_python.py

```diff
@@ -137,21 +141,21 @@ def run(selenium, img, ref_sobel, ref_laplacian, ref_canny):
         laplacian = cv.Laplacian(gray, cv.CV_8U, ksize=3)
         canny = cv.Canny(src, 100, 255)
 
-        np.compare_(ref_sobel, sobel, cv.IMREAD_GRAYSCALE)
-        np.compare_(ref_laplacian, laplacian, cv.IMREAD_GRAYSCALE)
-        np.compare_(ref_canny, canny, cv.IMREAD_GRAYSCALE)
+        comp.obj(ref_sobel, sobel, cv.IMREAD_GRAYSCALE)
```

Comment:
This `comp.obj` notation isn't all that beautiful. I wonder if I should have just unpickled the `PyodideHandle` directly to the wrapped object...

Reply:
> This `comp.obj` notation isn't all that beautiful. I wonder if I should have just unpickled the `PyodideHandle` directly to the wrapped object...

That would be nice :)

Reply:
PyodideHandle works, thanks a lot!

But I found that it timeouts at the teardown step. It doesn't make test fail but it shows error such as:

```py
opencv-python/test_opencv_python.py::calib3d_chessboard[chrome] PASSED                                                                             [100%]
Exception ignored in: <function PyodideHandle.__del__ at 0x7f03977015a0>
Traceback (most recent call last):
  File "/src/.docker_home/.local/lib/python3.10/site-packages/pytest_pyodide/decorator.py", line 96, in __del__
    self.selenium.run_js(
  File "/src/.docker_home/.local/lib/python3.10/site-packages/pytest_pyodide/runner.py", line 249, in run_js
    return self.run_js_inner(code, check_code)
  File "/src/.docker_home/.local/lib/python3.10/site-packages/pytest_pyodide/runner.py", line 349, in run_js_inner
    retval = self.driver.execute_async_script(wrapper % (code, check_code))
  File "/usr/local/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py", line 900, in execute_async_script
    return self.execute(command, {
  ...
urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='localhost', port=41453): Max retries exceeded with url: /session/4370d6910ba26ddd6567840b97dfc598/execute/async (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f039505b5e0>: Failed to establish a new connection: [Errno 111] Connection refused'))
```

My guess is selenium remote driver dies before deleting handle, do you have any ideas?

Reply:
Oh, never mind. There was a typo.